OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -153,12 +153,19 @@ List<CPos> CalculatePathToTarget(Actor self)
 				return pathFinder.FindBidiPath(fromSrc, fromDest);
 		}
 
-		public override IEnumerable<Target> GetTargets(Actor self)
+		public override Color TargetLineColor
 		{
-			if (inner != null)
-				return inner.GetTargets(self);
+			get
+			{
+				if (NextActivity != null)
+					return NextActivity.TargetLineColor;
+				return Color.Green;
+			}
+		}
 
-			return Target.None;
+		public override IEnumerable<Target> GetTargets(Actor self)
+		{
+			yield return Target;
```

Comment:
You removed the `inner.GetTargets(self)` above but didn't replace it here.  Is this intentional?